deepchem/feat/molecule_featurizers/molgan_featurizer.py

```diff
@@ -0,0 +1,243 @@
+import logging
+import numpy as np
+from deepchem.utils.typing import RDKitBond, RDKitMol, List
+from deepchem.feat.base_classes import MolecularFeaturizer
+
+logger = logging.getLogger(__name__)
+
+
+class GraphMatrix:
+  """
+    This is class used to store data for MolGAN neural networks.
```

Comment:
Spacing issue here on the docstring. Sometimes `yapf` does this in the automated conversion so you might have to just fix manually

Reply:
I have configured yapf in vs code to match google style and identation of 2. Weird that it does it. I will correct those manually.